src/app/page.tsx

```diff
@@ -118,18 +83,56 @@ export default function Home() {
             >
               Volunteer Now
             </LinkButton>
-          </Styles.ButtonDiv>
+          </div>
         </Styles.TextContainer>
       </Styles.TitleSection>
       <Styles.StatisticsSection>
-        {stats.map(stat => renderStat(stat))}
+        <Styles.StatInnerContainer>
+          {stats.map(stat => renderStat(stat))}
+        </Styles.StatInnerContainer>
       </Styles.StatisticsSection>
       <Styles.HowYouCanHelpContainer style={{ background: COLORS.background }}>
-        <H2 $color={COLORS.blueMid}>
-          HOW <span style={{ color: COLORS.goldMid }}>YOU</span> CAN HELP
+        <H2 $color={COLORS.blueMid} $fontWeight={400}>
+          HOW <strong style={{ color: COLORS.goldMid }}>YOU</strong> CAN HELP
         </H2>
         <Styles.ServicesDiv>
-          {services.map(Service => renderService(Service))}
+          <Styles.ServiceContainer>
```

Comment:
is there a reason why we got rid of the `renderService` approach?

Reply:
mostly because it's bad practice to turn a string to HTML tags, as the name implies, this is dangerously and it's best to not do that if we can help it

Reply:
but in hindsight, i can bring it back using TSX instead of strings